src/Validation/Validator.php

```diff
@@ -556,9 +573,13 @@ protected function _processRules($field, ValidationSet $rules, $data, $newRecord
                 continue;
             }
 
-            $errors[$name] = $message;
             if (is_string($result)) {
                 $errors[$name] = $result;
+            } elseif ($this->_useI18n) {
+                $args = $rule->get('pass');
+                $errors[$name] = __d($this->_validationDomain, $name, $this->_translateArgs($args));
```

Comment:
@lorenzo  For me personally most commonly repeated and cumbersome are parameterized rules related to file uploads: fileSize, extensions, image dimensions etc. Given that the arguments for these validators could be different for each table I would have to specify the message each time as `'message' => __d('domain', 'fileSize', '5MB')` , `'message' => __d('domain', fileSize', '10MB')` and have translation "Max allowed file size is: {0}" for "fileSize". With the proposed change I only have to name the rule "fileSize" when adding validation rule.